src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -641,23 +640,25 @@ void jit_store_emitter::emit_isa(const int in_vec_idx, const Xbyak::Reg64 &reg_d
     }
     using Vmm = typename conditional3<isa == cpu::x64::sse41, Xmm, isa == cpu::x64::avx2, Ymm, Zmm>::type;
 
-    int data_idx = in_vec_idx;
+    original_data_idx = in_vec_idx;
```

Comment:
As far as I understood, the primary purpose of `original_data_idx` is to indicate that `data_idx` will be corrupted and it's update is required ( numerous `if (data_idx == original_data_idx)` statements). So why don't we store a boolean variable with a descriptive name instead? like `data_reg_update_required`? Then all the `if` statements will look like `if (data_reg_update_required)` which is more readable. What do you think?

Reply:
I agree, have updated, thank you!